lib/amazon_ssa_support/ssa_queue.rb

```diff
@@ -12,13 +12,13 @@ class SsaQueue
     def initialize(args)
       @extractor_id       = args[:extractor_id]
       @ssa_bucket_name    = args[:ssa_bucket]
-      @ssa_region         = args[:region] || DEFAULT_REGION
+      @ssa_region         = args[:region]
       @request_queue_name = args[:request_queue] || DEFAULT_REQUEST_QUEUE
       @reply_queue_name   = args[:reply_queue] || DEFAULT_REPLY_QUEUE
       @reply_prefix       = args[:reply_prefix] || DEFAULT_REPLY_PREFIX
 
-      unless ssa_bucket_name && extractor_id
-        raise ArgumentError, "extractor_id & ssa_bucket_name must have to be specified."
+      unless ssa_bucket_name && ssa_region
+        raise ArgumentError, "bucket_name & region must have to be specified."
```

Comment:
@hsong-rh The way it's phrased the word "have" should be removed, so that it reads "bucket_name and region must be specified".

Actually, to be consistent with the other error messages, it should probably be changed to, "Bucket and region must be specified".

Sorry if I'm being nitpicky.